lib/modules/datasource/packagist/schema.ts

```diff
@@ -1,28 +1,36 @@
 import { z } from 'zod';
+import { logger } from '../../../logger';
 import { api as versioning } from '../../versioning/composer';
 import type { Release, ReleaseResult } from '../types';
 
-export const ComposerRelease = z.object({
-  version: z
-    .string()
-    .refine((v) => versioning.isSingleVersion(v), 'Invalid version'),
-  homepage: z.string().nullable().catch(null),
-  source: z
-    .object({
-      url: z.string(),
-    })
-    .transform((x) => x.url)
-    .nullable()
-    .catch(null),
-  time: z.string().nullable().catch(null),
-});
+const Version = z
+  .string()
+  .refine((v) => versioning.isSingleVersion(v), 'Invalid version');
```

Comment:
Maybe this mistake was made earlier but we should not be using any `versioning` within `lib/modules/datasource/*/*`. Versioning filtering should be done by the datasource index/dispatcher and not hardcoded into datasource implementations

Reply:
Example search showing that this is unique:

![image](https://user-images.githubusercontent.com/6311784/212719138-47736d2f-ea41-4ac9-9ac9-6344b7f5ca14.png)

Reply:
But we still need to use version comparison and validity check for deciding on most fresh `homepage` and `sourceUrl` (except for unstable releases like `dev-main`)?

Reply:
Actually, I think we can safely assume the first release to be the latest one

Reply:
Yes, use the first